employees/views.py

```diff
@@ -255,3 +259,29 @@ def form_valid(self, form: AdminReportForm) -> HttpResponseRedirectBase:
         self.object.editable = True
         self.object.save()
         return super().form_valid(form)
+
+
+@method_decorator(login_required, name="dispatch")
+class ExportUserReportView(LoginRequiredMixin, DetailView):
+    model = CustomUser
+
+    def render_to_response(self, context: dict, **response_kwargs: Any) -> HttpResponse:
+        author = super().get_object()
+        response = HttpResponse(content_type="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+        response["Content-Disposition"] = f'attachment; filename="{author.email}-reports.xlsx"'
```

Comment:
If you want to keep everything in constants, why not those two values? Event more that they are reused below.

Reply:
Done

Reply:
Where? I can't see it :)

Reply:
I am sorry I added it only to one view